src/collection.js

```diff
@@ -103,7 +103,7 @@ function createUUIDSchema() {
 }
 
 function markStatus(record, status) {
-  return Object.assign({}, record, {_status: status});
+  return Object.assign({}, record, _status: status);
```

Comment:
As described in the issue description, every `Object.assign()` should be removed